src/components/SButton.vue

```diff
@@ -0,0 +1,15 @@
+<script>
+export default {
+  name: 'SButton'
+}
+</script>
+
+<template>
+    <button>
+        <slot name="value"/>
+    </button>
```

Comment:
If you leave your slot with no `name` then you can just pass whatever between the components tags and it will be a default slot.

I.e.

```
<button>
   <slot />
</button>
```

```
<SButton>
    This is the slot
</SButon>
```